src/Util/Temporal.php

```diff
@@ -217,13 +217,13 @@ public static function getDateTimeField(
 		// First day of the week (0 = Sunday)
 		$firstDay = PConfig::get(local_user(), 'system', 'first_day_of_week', 0);
 
-		$lang = substr(L10n::getBrowserLanguage(), 0, 2);
+		$lang = substr(L10n::getCurrentLang(), 0, 2);
 
 		// Check if the detected language is supported by the picker
 		if (!in_array($lang,
 				["ar", "ro", "id", "bg", "fa", "ru", "uk", "en", "el", "de", "nl", "tr", "fr", "es", "th", "pl", "pt", "ch", "se", "kr",
 				"it", "da", "no", "ja", "vi", "sl", "cs", "hu"])) {
-			$lang = Config::get('system', 'language', 'en');
+			$lang = 'en';
```

Comment:
Why don't we use the config value anymore?

Reply:
If the system language isn't among the list the picker handles, then this check is useless. So far we haven't had a problem because the picker supports all the languages Friendica nodes are configured with that we know of.